blueocean-dashboard/src/main/js/components/karaoke/components/Step.jsx

```diff
@@ -139,7 +134,7 @@ export class Step extends Component {
             liveFormat={t('common.date.duration.format', { defaultValue: 'm[ minutes] s[ seconds]' })}
             hintFormat={t('common.date.duration.hint.format', { defaultValue: 'M [month], d [days], h[h], m[m], s[s]' })}
         />);
-        return (<div className={logConsoleClass} key={this.pager.currentLogUrl}>
+        return (<div className={logConsoleClass}>
```

Comment:
@cliffmeyers I am not 100% sure but I think that has been there because of the effect it has on rendering,  I remember adding it when using the satges and steps and the steps had been from the old stage, but I guess that is not needed anymore then ;)